.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
 
 before_install:
   - gem install bundler
-  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
+  - google-chrome-stable --headless -no-sandbox --disable-gpu --remote-debugging-port=9222 http://localhost &
```

Comment:
Should this be `--no-sandbox`? I'm having a hard time tracking down the docs for the flags.